### PR TITLE
Add a version command

### PR DESCRIPTION
**Pull request by trisberg**
- change the default for function version to be 0.0.1 instead of v0001

### CODE REVIEW

File: riff [hunk 1]
```diff
@@ -1,17 +1,22 @@
 #!/bin/bash
 
+RIFF_VERSION=0.0.1
```
Comment (markfisher) [inline riff]: shouldn't this one be 0.0.2? (or will that be a single commit before the release?)

Comment (trisberg) [reply riff]: I'll update that when we release

Comment (markfisher) [reply riff]: cool

**Comment by markfisher** [comment]
LGTM; if you want to handle it (with that one change)

**Pull request merged**